src/components/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,58 @@
+import SideBarItem from "./SideBarItem";
+import { Box, Flex } from "@chakra-ui/react";
+// import './style.css';
+
+const SideBar = () => {
```

Comment:
At present, there is no problem with this naming, but if there is a new sidebar in the future, there may be conflicts. In that case, you can add some words before the sidebar to indicate what the sidebar is used for